packages/config/config/devices/0x0086/dsb28.json

```diff
@@ -35,210 +35,181 @@
+		},
+		"101[0x0100]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_watt_clamp1"
+		},
+		"101[0x0200]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_watt_clamp2"
+		},
+		"101[0x0800]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_kwh_clamp1"
+		},
+		"101[0x1000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_kwh_clamp2"
+		},
+		"101[0x10000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_v_clamp1"
+		},
+		"101[0x20000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_v_clamp2"
+		},
+		"101[0x80000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_amp_clamp1"
+		},
+		"101[0x100000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group1_amp_clamp2"
+		},
+		"102[0x01]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_kwh_wholehem"
+		},
+		"102[0x02]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_watt_wholehem"
+		},
+		"102[0x04]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_v_wholehem"
+		},
+		"102[0x08]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_amp_wholehem"
+		},
+		"102[0x0100]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_watt_clamp1"
+		},
+		"102[0x0200]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_watt_clamp2"
+		},
+		"102[0x0800]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_kwh_clamp1"
+		},
+		"102[0x1000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_kwh_clamp2"
+		},
+		"102[0x10000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_v_clamp1"
+		},
+		"102[0x20000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_v_clamp2"
+		},
+		"102[0x80000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_amp_clamp1"
+		},
+		"102[0x100000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group2_amp_clamp2"
+		},
+		"103[0x01]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_kwh_wholehem"
+		},
+		"103[0x02]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_watt_wholehem"
+		},
+		"103[0x04]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_v_wholehem"
+		},
+		"103[0x08]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_amp_wholehem"
+		},
+		"103[0x0100]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_watt_clamp1"
+		},
+		"103[0x0200]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_watt_clamp2"
+		},
+		"103[0x0800]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_kwh_clamp1"
+		},
+		"103[0x1000]": {
+			"$import": "templates/aeotec_template.json#auto_report_group3_kwh_clamp2"
+		},
+		"103[0x10000]": {
```

Comment:
Please make bitmasks byte-aligned (`0x`, followed by 2, 4, 6 or 8 digits), even if it means you have leading zeroes

Reply:
Do you mean that this should be [0x001000]?